### PR TITLE
show selectors

### PR DESCRIPTION
**Pull request by inajob**
Signed-off-by: inajob <kinadu@zlab.co.jp>

**Pull Request check list**

- [x] Commit conforms to CONTRIBUTING.md?
- [x] Proper tests/regressions included?
- [ ] Documentation updated?

**Affected functionality**
`spire-server agent show` command

**Description of change**

I added show selectors feature in `spire-server agent  show`.

This feature was implemented by https://github.com/spiffe/spire/pull/1098 . But it has gone now.

The example of output is below.

```
Found an attested agent given its SPIFFE ID

SPIFFE ID         : spiffe://example.org/spire/agent/k8s_psat/demo-cluster/2aec14c3-1e63-4d57-a85c-dc128fd81ca1
Attestation type  : k8s_psat
Expiration time   : 2021-03-04 02:58:21 +0000 UTC
Serial number     : 136426352658711670953411608294207618767

Selectors     : k8s_psat:cluster:demo-cluster
Selectors     : k8s_psat:agent_ns:spire
Selectors     : k8s_psat:agent_sa:spire-agent
Selectors     : k8s_psat:agent_pod_name:spire-agent-vprvn
Selectors     : k8s_psat:agent_pod_uid:c08194af-3c62-44df-b621-2351bdd40713
Selectors     : k8s_psat:agent_node_ip:192.168.49.2
Selectors     : k8s_psat:agent_node_name:minikube
Selectors     : k8s_psat:agent_node_uid:2aec14c3-1e63-4d57-a85c-dc128fd81ca1
```

**Which issue this PR fixes**

https://github.com/spiffe/spire/issues/1073

### CODE REVIEW

File: cmd/spire-server/cli/agent/show.go [hunk 1]
```diff
@@ -62,6 +62,11 @@ func (c *showCommand) Run(ctx context.Context, env *common_cli.Env, serverClient
 		return err
 	}
 
+	if agent.Selectors != nil {
+		for _, s := range agent.Selectors {
```
Comment (azdagron) [inline cmd/spire-server/cli/agent/show.go:66]: Can you move this into `printAgents`? Can you also align the `:` for each line being printed?

**Review by azdagron — COMMENTED** [review]
Oh wow. Thanks @inajob. I can't believe this was lost when we transitioned the CLI to the new APIs. One day we'll have better integration testing around the CLI to prevent this kind of thing.

I've left a few minor comments.

File: cmd/spire-server/cli/agent/show.go (hunk 1 above)
Comment (inajob) [reply cmd/spire-server/cli/agent/show.go:66]: `printAgents` is also used by `spire-server agent list` command.
I think this feature should be only activate in `spire-server agent show` command.

File: cmd/spire-server/cli/agent/show.go [hunk 2]
```diff
@@ -62,6 +62,11 @@ func (c *showCommand) Run(ctx context.Context, env *common_cli.Env, serverClient
 		return err
 	}
 
+	if agent.Selectors != nil {
```
[the remark this answers is not in the document]
Comment (inajob) [reply cmd/spire-server/cli/agent/show.go]: thx!
Removed. https://github.com/spiffe/spire/pull/2133/commits/9dc0016b55a5a885cf25227a4dc1ba3c0e50b9ed

File: cmd/spire-server/cli/agent/show.go (hunk 1 above)
Comment (inajob) [reply cmd/spire-server/cli/agent/show.go]: Aligned to other items. https://github.com/spiffe/spire/pull/2133/commits/9dc0016b55a5a885cf25227a4dc1ba3c0e50b9ed

Comment (azdagron) [reply cmd/spire-server/cli/agent/show.go]: > `printAgents` is also used by...
Oh yep. You are right. Thanks!

**Review by azdagron — APPROVED** [review]

**Comment by azdagron** [comment]
Thanks @inajob !

**Comment by azdagron** [comment]
Thanks @inajob !

**Pull request merged**